packages/console/src/containers/ConsoleContent/index.module.scss

```diff
@@ -16,7 +16,7 @@
   }
 
   // App Insights wrapper on cloud env prevents the CSS assignment to direct children
-  .appInsightsWrapper > * {
+  [class='appInsightsWrapper'] > * {
```

Comment:
just curious didn't this test before?

Reply:
Manually added a div wrapper in the Chrome console and tested and bypassed the condition of creating `appInsights` instance. Seems this does not reproduce the same environment running on the cloud.

Reply:
cool. remember to repro locally as we have the connection string now